### PR TITLE
tide install.sh

### LINKED ISSUE #25: Tidy install.sh
- [x] Replace `apt-get` with `apt` on `install.sh`
 - [x] Remove extra empty lines.
 - [x] Improve comments.

### PR DESCRIPTION
**Pull request by d78ui98**
- [x]  Replace apt-get with apt on `install.sh`
- [x]  Other improvments `install.sh`
- [x]  Improvments in `Vagrantfile`

closes #25

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -1,61 +1,60 @@
...
 
-#Installing additional software
-apt-get --yes install edubuntu-desktop
-apt-get --yes install ubuntu-edu-preschool ubuntu-edu-primary ubuntu-edu-secondary ubuntu-edu-tertiary
+# Installing additional software
+apt --yes install ubuntu-edu-preschool ubuntu-edu-primary ubuntu-edu-secondary ubuntu-edu-tertiary
 
-#Installing ltsp-manager
+# Installing ltsp-manager
 add-apt-repository ppa:ts.sch.gr
```
Comment (akash0x53) [inline install.sh]: doesn't it ask confirmation while adding repo?

**Review by dashohoxha — APPROVED** [review]
You can fix/improve it later.
Akash can also review and approve your changes.

File: install.sh [hunk 2]
```diff
@@ -1,61 +1,64 @@
...
-apt-get --yes install ubuntu-edu-preschool ubuntu-edu-primary ubuntu-edu-secondary ubuntu-edu-tertiary
+# Installing additional software
+apt --yes install ubuntu-edu-preschool ubuntu-edu-primary ubuntu-edu-secondary ubuntu-edu-tertiary
 
-#Installing ltsp-manager
-add-apt-repository ppa:ts.sch.gr
-apt-get --yes update
-apt-get --yes install ltsp-manager
+# Installing ltsp-manager
+echo | add-apt-repository ppa:ts.sch.gr
```
Comment (akash0x53) [inline install.sh:35]: "echo yes" you mean?

Comment (d78ui98) [reply install.sh:35]: it asks for enter. We can use `echo enter` as well.
But this is cleaner So I used this one.
I am attaching screenshot for output of both - https://image.ibb.co/nA4Xsy/echoaddrepo.png

Comment (akash0x53) [reply install.sh:35]: okay.

**Review by akash0x53 — APPROVED** [review]

**Pull request merged**